### PR TITLE
Upgrade deps

### PR DESCRIPTION
**Pull request by mfleader**
## Changes introduced with this PR

- [X] upgrade dependencies to fix CVE

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,16 +3,17 @@ module go.flow.arcalot.io/podmandeployer
 go 1.21
 
 require (
-	github.com/docker/docker v24.0.7+incompatible
-	github.com/docker/go-connections v0.4.0
+	github.com/docker/docker v25.0.2+incompatible
+	github.com/docker/go-connections v0.5.0
```
Comment (dbutenhof) [inline go.mod:7]: The podman deployer uses docker types? Interesting...

**Review by dbutenhof — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (webbnh) [reply go.mod:7]: It gives new depth to the notion that Podman is plug-compatible with Docker....

**Review by webbnh — APPROVED** [review]
👍

**Review by webbnh — APPROVED** [review]
👍

**Pull request merged**